flow_prompt/exceptions.py

```diff
@@ -13,6 +13,9 @@ class FlowPromptIsnotFoundException(FlowPromptException):
 class BehaviourIsNotDefined(FlowPromptException):
     pass
 
+class ConnectionLostException(FlowPromptException):
```

Comment:
ideal to rename to  ConnectionLostError, same as FlowPromptError, and other error classes

Reply:
it's more minor